version.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+const version = "v0.0"
```

Comment:
I'd suggest using `1.0.0` similar to how Icinga 2 does it in its VERSION file.

Reply:
1. This one intentionally complies to `git describe` for packaging.
2. There's no 1.0, yet.